lib/trollop.rb

```diff
@@ -61,7 +61,11 @@ class Parser
   ##  ignore options that it does not recognize.
   attr_accessor :ignore_invalid_options
 
+  ## Settings for the entire option parser that apply to all given options.
+  attr_reader :settings
```

Comment:
Leaning away from this feature.
Although, I guess we do have some global behavior embedded in this gem